docs/usage.md

```diff
@@ -144,6 +144,24 @@ Then pass this to `osv-scanner` with this:
 osv-scanner --lockfile osv-scanner:/path/to/osv-scanner.json
 ```
 
+## Specify Package(s) to Exclude from OSV Query
```

Comment:
Thank you for updating the documentation for the --exclusions flag. I appreciate you including this as part of your PR. 

Our GitHub Page theme automatically wraps text. Each paragraph should be included in a single line. This makes future changes easier and is more consistent with the majority of the documentation (though there are a few places where I need to fix this).

Reply:
Hi @hayleycd did you need me to update the paragraphs to be a single line to fit in with your Github Pages theme? Or it is fine to leave the paragraph wrapped?

Reply:
I added a commit. Docs are good to go now.

Reply:
Thanks @hayleycd, I have squashed this into the main commit.